faqs/galaxy/datasets_dataset_downloading_incompletely.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Dataset downloading incompletely?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Try using the Google Chrome web browser. Sometimes Chrome itself better supports continuous data transfers.
+2. Use the command-line option instead. The data may really be too large to download OR your connection is slower. This can also be a faster way to download multiple datasets plus ensure a complete transfer (small or large data).
```

Comment:
we should probably add an FAQ about "the command line option", if we are going to suggest it here